source/_integrations/fronius.markdown

```diff
@@ -88,6 +98,58 @@ Recommended energy dashboard configuration for meter location in feed in path (`
 - `Battery systems` aren't supported directly. Use [Template](/integrations/template) to split and invert negative values of `Power battery` entity (from power_flow) for charging power (W) and positive values for discharging power (W). Then use [Riemann sum](/integrations/integration/) to integrate each into energy values (kWh).
 - For `Devices` use the Ohmpilots `Energy consumed` entity.
 
+### Consumption path
+
+Recommended energy dashboard configuration for meter location in consumption path (`Meter location: 1`):
+
+This configuration has been possible thanks to [@GianlucaCollot work](https://github.com/GianlucaCollot), honors and credit go to him as I have just reproduced his work and documented it here :)
+
+First you need to create the power entities in your `configuration.yaml`. Make sure to replace the name of the sensor, you may only need to update the IP on the end if it was auto-detected:
+
+```yaml
+template:
+  - sensor:
+    # Positive values are from grid
+    - name: "power_bought"
+      unit_of_measurement: W
+      state: >-
+        {{ max(states('sensor.power_grid_fronius_power_flow_0_XXX_XXX_XXX_XXX') | float, 0) }}
+      device_class: power
```

Comment:
This is a bug IMHO and not a documentable solution.

It is wrapping an existing sensor to solve an issue (as it seems).

Reply:
The "Power Grid" entity as provided from Fronius API is positive on import and negative on export. Hence the `max(state, 0)` and `max(0- state, 0)` to split it up into two entities - import and export to be used in energy (after integration).